internal/runners/auth/auth.go

```diff
@@ -31,11 +30,12 @@ func NewAuth(prime primeable) *Auth {
 }
 
 type AuthParams struct {
-	Token       string
-	Username    string
-	Password    string
-	Totp        string
-	Interactive bool
+	Token          string
+	Username       string
+	Password       string
+	Totp           string
+	Interactive    bool
+	NonInteractive bool
```

Comment:
What is the relationship between `Interactive` and `NonInteractive`? Are they mutually exclusive? If so, perhaps these two fields can be replaced by an `Interactivity` enum.

Reply:
They are not mutually exclusive. Interactive in this case basically means "interact via my terminal" and non-interactive means "do not interact with me in ANY way; browser or otherwise".